tests/ctst/world/Zenko.ts

```diff
@@ -591,7 +697,7 @@ ${JSON.stringify(policy)}\n${err.message}\n`);
      * @param {object} param - an object with a key and a value
      * @returns {undefined}
      */
-    addCommandParameter(param: AwsCliObjectParameters): void {
+    addCommandParameter(param: Record<string, unknown>): void {
```

Comment:
I don't get why this change is needed as the previous type `AwsCliParameters` was already embedding everything? ANything not in this type means something not supported.

Reply:
Not sure I understand what you mean?
```
interface AwsCliObjectParameters {
    [key: string]: number | string | undefined | object;
```
Why is this better than a `Record<string, unknown>` ?

Reply:
The initial type didn't come from CTST, it was declared at the top of the Zenko world file just to facilitate typing. I think you might be confusing `AwsCliObjectParameters` which was used for this typing and `AwsCliParameters` which is used within CTST specifically, but also `AwsCliOptions` which IS the type that embeds all possible parameters (generated from API). 

If I add a `[key: string]: string` to the generator for the `AwsCliOptions` type, I can use it as parameter type here, what do you think? 
While at it, maybe I could replace the `AwsCliParameters` occurences/functionality in CTST with `AwsCliOptions`, but it might be a bit of work (and out of scope); atleast worth a ticket?

Reply:
Please see https://github.com/scality/Zenko/pull/1818#discussion_r1184762254 for the initial comment, I was commenting the wrong line, it indeed does not make sense for this command, but it does for `getCommandParameters`.

The new point here is that we can rework this type you added in CTST to be more flexible, to avoid casting here